frontend/src/pages/order/libs/components/order-form.tsx

```diff
@@ -0,0 +1,47 @@
+import { Form } from '~/libs/components/components.js';
+import { orderCreateRequestBody } from '~/packages/orders/orders.js';
+import { type OrderCreateRequestDto } from '~/packages/orders/orders.js';
+
+import { CREATE_ORDER_DEFAULT_PAYLOAD } from './libs/constants.js';
+import { getOrderFormFields } from './libs/fields.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  isDisabled?: boolean;
+  truckId: number;
+  children: JSX.Element;
+  onSubmit: (payload: OrderCreateRequestDto) => void;
+  onStartLocationChange: (
+    place: google.maps.LatLngLiteral,
+    address: string,
+  ) => void;
+  onEndLocationChange: (
+    place: google.maps.LatLngLiteral,
+    address: string,
+  ) => void;
```

Comment:
add a separate type for this fn, and reuse in onStartLocChange, onEndLocationChange